src/99-appendices/06-meg-file-formats.md

```diff
@@ -52,21 +52,44 @@ To learn more about CTF’s data organization:
 
 ## Neuromag/Elekta/MEGIN
 
-Neuromag/Elekta/MEGIN data and Tristan Technologies BabyMEG data is stored with
-file extension `.fif`. The digitized positions of the head points are saved
-inside the fif file along with the MEG data, with typically no `_headshape`
-file.
+Neuromag/Elekta/MEGIN and Tristan Technologies BabyMEG data is stored as
+FIFF files with the extension `.fif`. The digitized positions of the head
+points are saved inside the fif file along with the MEG data, with typically no
+`_headshape` file.
 
 ```Text
 sub-<label>[_ses-<label>]_task-<label>[_run-<index>]_meg.fif
 ```
 
-Note that we do not provide specifications for cross-talk and
-fine-calibration matrix files in the current version of BIDS.
+### Cross-talk and fine-calibration files
 
-Example:
+Raw FIFF files typically need to be processed using Maxwell filtering
+(signal-space separation, SSS) to make the data usable.
+To this end, two specific files are needed:
+The *cross-talk* file, and the *fine-calibration* file,
+both of which are produced by the MaxFilter software
+and the work of the Neuromag/Elekta/MEGIN engineers during maintenance of the MEG acquisition system.
+Both files are thus specific to the site of recording and may change in the
+process of regular system maintenance.
+
+In BIDS, the cross-talk and fine-calibration files are shared unmodified,
+but with BIDS file naming convention and by using the `acq` entity.
+
+-   cross-talk file template: `[sub-<label>][_ses-<label>][_]acq-crosstalk_meg.dat`
+-   fine-calibration file template: `[sub-<label>][_ses-<label>][_]acq-finecalib_meg.fif`
```

Comment:
whall we call this calibration? so calibration_meg.fif

Reply:
@hoechenberger suggested to use something shorter than `finecalibration`, so I changed to `finecalib` ... but I have no strong feeling towards one or the other.

Reply:
Let's use `calibration` then, I have no strong feelings tbh!

Reply:
- stefan: finecalibration
- richard: finecalib
- alex: calibration

:shrug: 

I'd say let's stick with the long name. I always like explicit names, because users tend to get confused ...

Reply:
I thought the beauty of `finecalib` was that it's exactly the same number of characters as `crosstalk` ;)

```
acq-crosstalk
acq-finecalib
```

**Isn't that pretty??** 😅

Reply:
@sappelhoff I think "finecalibration" is really unnecessarily long… but idk!! I just changed everything to `calibration` but feel free to revert

Reply:
Maybe "calibration" is a good compromise here then, after all

Reply:
yes, I think that's fine -> thank you :-)

Reply:
> yes, I think that's fine -> thank you :-)

It's not just **fine** – it's **fine-calibrated** !!!